src/package.cr

```diff
@@ -67,6 +67,20 @@ module Shards
       raise ex
     end
 
+    def install_executables
+      if installed? && spec.executables.any?
+        bin_path = File.join(Dir.current, "bin")
+        Dir.mkdir_p(bin_path)
+
+        spec.executables.each do |name|
+          Shards.logger.debug "Install bin/#{name}"
+          source = File.join(resolver.install_path, "bin", name)
+          dest = File.join(bin_path, name)
+          FileUtils.cp(source, dest)
```

Comment:
Why not symlink?

Reply:
Because `bin` should assume to have binaries? Thus, to prepare a package for example, I can `cp bin/foo dist/bin/` rather than `cp $(realpath bin/foo) dist/bin/` or digging `cp lib/foo/bin/foo dist/bin` because `bin/foo`.

Maybe we could hardlink?